repo/packages/K/kafka/0/marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "kafka",
+  "cpus": 0.5,
+  "mem": 256,
+  "instances": 1,
+  "cmd": "java -jar kafka-mesos*.jar scheduler --master=zk://master.mesos:2181/mesos --zk=master.mesos:2181 --api=http://localhost:$PORT0",
```

Comment:
Scheduler is listening for HTTP connections on that port.
It serves 2 tasks:
- provide REST for CLI;
- distribute binaries for executor;

Of course the executor could be also download from the outside, but, IMHO, it would be better to download it from scheduler, because it is the same jar file and is in local network.